client/scripts/directives/field.js

```diff
@@ -46,6 +46,21 @@ require('../app').directive('field', /* @ngInject */function ($q) {
       field.type = $attrs.type
       field.required = angular.isDefined($attrs.required)
       field.readonly = 'readonly' in $attrs ? (angular.isDefined($attrs.readonly) ? $parse($attrs.readonly)($scope.$parent) : true) : false
+
+      if ('disabled' in $attrs) {
+        if (angular.isDefined($attrs.disabled)) {
+          var disabledGetter = $parse($attrs.disabled).bind(null, $scope.$parent)
+          field.disabled = disabledGetter()
```

Comment:
This can be removed, the watcher on the next line will be executed on next tick and will override the value